models/linkages/R/write_restart.LINKAGES.R

```diff
@@ -212,16 +214,19 @@ write_restart.LINKAGES <- function(outdir, runid, start.time, stop.time, setting
       new.ntrees[s] = sample(size = 1, x = 50:150)
     } 
   }
+  
+  if(sum(new.ntrees) > 198) new.ntrees <- round((new.ntrees / sum(new.ntrees)) * runif(1,160,195))
+  
```

Comment:
This seems like a hack -- could you at least provide a comment explaining the rational for this and whether it's intended to be temporary or not

Reply:
So, this is to enforce stem density in linkages. I could estimate it with sda maybe? or have it as a parameter like carrying capacity?

Reply:
In the short term I wouldn't add stem density to the SDA since in the current example we don't have data to estimate it, plus you might still need some constraint if the SDA itself pushes the density too high. I'll leave the long term solution to you (e.g. carrying capacity or other option) but in the short term I think I'd be happy if you just added some comments here explaining.